### ISSUE TITLE
Mark: Brand mark

### ISSUE THREAD

**Issue by louismaximepiton** [Orange-OpenSource/Orange-Boosted-Bootstrap#1504]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

Use black on white or white on black to have a better contrast. Some more specifications will come on the design system soon.

![image](https://user-images.githubusercontent.com/91960143/187445404-9d9ad615-9f25-43a4-b705-d6f0020cc962.png)

### Motivation and context

Help people to use a good color contrast.
Be branded before the brand itself.
